### ISSUE TITLE
requestBody with oneOf schemas produces an error

### ISSUE THREAD

**Issue by ahannigan** [readmeio/api-explorer#104]
When using a valid OpenAPI 3.0.0 specification with a requestBody that contains a oneOf schema like below:

```yaml
requestBody:
  content:
    application/json:
      schema:
        oneOf:
          - $ref: '#/components/schemas/One'
          - $ref: '#/components/schemas/Two'
          - $ref: '#/components/schemas/Three'
```

the following error is produced:

`Unsupported field schema for field root: Unknown field type undefined.`

followed by the oneOf JSON blob, rather than rendering the available request bodies.

**Comment by uppal101** [comment]
Unfortunately the library in which we are using does not have the support for this yet once it does we will support it. https://github.com/mozilla-services/react-jsonschema-form/pull/417

**Comment by spifd** [comment]
Any update supporting this? Apparently, react-jsonschema-form still doesn't support it. This is a strong limitation when handling advanced JSON structure in API.

**Comment by epicfaace** [comment]
We've recently merged in a PR that will support it in react-jsonschema-form, so expect it in the next release: mozilla-services/react-jsonschema-form#1133

**Comment by ekeren** [comment]
Hey, any idea when is the next release?

**Comment by epicfaace** [comment]
The above PR has already been released in [1.2.1](https://github.com/mozilla-services/react-jsonschema-form/releases/tag/v1.2.1), @ekeren

**Comment by domharrington** [comment]
This is now fixed: https://preview.readme.io/?selected=swagger-files%2Fone-of.json
